### DISCUSSION TITLE
Tutotial 16.3.4 "add an ability to kill the bots"

### DISCUSSION THREAD

**Discussion by realroot2185** [FyroxEngine/Fyrox#667] [General]
> [Conclusion](https://fyrox-book.github.io/tutorials/fps/tutorial-3/fps-tutorial-3.html#conclusion)
> 
> In this tutorial part we've added bots with animation and simple AI. In the next tutorial we'll add an ability to kill the bots.

Is that part coming some day?

**Comment by mrDIMAS** [comment]
Can't say for sure, writing tutorials is hard and quite time consuming, so the answer to your question is "probably".
